### DISCUSSION TITLE
Russian language support

### DISCUSSION THREAD

**Discussion by mrkhachaturov** [totaldebug/atomic-calendar-revive#1241] [Feature Requests] [answered]
According to the 2019 edition of Ethnologue, an American-produced language reference guide, there are approximately 258 million Russian speakers worldwide. 153.6 million of those speak Russian as a first language, helping to rank it as number seven in the world.

**Comment by marksie1988** [comment, marked as answer]
This has been merged to the latest release :)

**Reply by mrkhachaturov** [reply]
Thank you  so much :) 
Updated to latest version.
